web/modules/custom/dcamp_sessions/src/Session.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\dcamp_sessions;
+
+use Drupal\Component\Utility\Xss;
+
+/**
+ * Class to interact with session data
+ */
+class Session {
+
+  /**
+   * Positions in sessionsGathered array
+   */
+  const NAME = 2;
+  const DRUPAL_ORG = 3;
+  const PHOTO = 4;
+  const TWITTER = 5;
+  const BIO = 6;
+  const SESSION_TYPE = 7;
+  const SESSION_LEVEL = 8;
+  const LANGUAGE = 9;
+  const TITLE= 10;
+  const DESCRIPTION = 11;
+
+  /**
+   * @var string[]
+   */
+  private $sessionGathered;
+
+  public function __construct( $session_gathered) {
+    $this->sessionGathered = $session_gathered;
+  }
+
+  public function getAuthorsName() {
+    return Xss::filter($this->sessionGathered[$this::NAME]);
```

Comment:
I think that this is not needed, @isholgueras, because Twig escapes by default all variables. Can you verify? I fear that we may be double escaping things.